modules/vocabularies/src/main/java/ca/sickkids/ccm/lfs/vocabularies/internal/OboParser.java

```diff
@@ -51,6 +51,7 @@
 @Component(
     service = SourceParser.class,
```

Comment:
```suggestion
    service = SourceParser.class,
    scope=ServiceScope.PROTOTYPE,
```

I wonder if just adding this line makes it work, without the need to change anything else.

Reply:
This also seems to work, though I'm more in favour of `InheritableThreadLocal` since it's also the approach currently being used by `ca.sickkids.ccm.lfs.vocabularies.internal.OwlParser` (i.e. we should either adopt it in both cases and forego `InheritableThreadLocal` or use `InheritableThreadLocal` in both cases).